layouts/shortcodes/committers.html

```diff
@@ -0,0 +1,26 @@
+{{ $urlPre := "https://projects.apache.org" }}
+{{ $path :=  "/json/foundation" }}
+{{ $group :=  getJSON $urlPre $path "/groups.json"}}
+{{ $name := getJSON $urlPre $path "/people_name.json" }}
+{{$orgnisations := getJSON "data/teamdata.json" }}
+
+<table class="tableblock frame-all grid-all stretch">
```

Comment:
I don't see `frame-all` and `grid-all` CSS classes in our CSS, are these needed or are they missing?

Reply:
These classes already existed in /layout/shortcodes/table.html.

Reply:
Yeah looks like that was a mistake there, I don't see those classes defined in CSS so we might just as well remove them now.